### PR TITLE
SDK-573: Fix sandbox code to allow multiple derivations of an attribute

### PR DESCRIPTION
**Pull request by bucky-boy**
We also need a little change in the SDK sandbox code.
I've explained in the SDK-573 ticket.

### CODE REVIEW

File: yoti-sdk-sandbox/src/test/java/com.yoti.api.client.sandbox/profile/request/YotiTokenRequestTest.java [hunk 1]
```diff
@@ -47,13 +46,19 @@
...
     @Test
-    public void shouldAddAnAttribute() {
+    public void shouldAddAttributes() {
+        SandboxAttribute otherAttribute = SandboxAttribute.builder()
+                .withName("otherAttributeName")
+                .build();
+
         YotiTokenRequest yotiTokenRequest = YotiTokenRequest.builder()
                 .withAttribute(SOME_ATTRIBUTE)
+                .withAttribute(otherAttribute)
```
Comment (sidibos) [inline yoti-sdk-sandbox/src/test/java/com.yoti.api.client.sandbox/profile/request/YotiTokenRequestTest.java]: Will the derived attributes have anchors?

File: yoti-sdk-sandbox/src/test/java/com.yoti.api.client.sandbox/profile/request/YotiTokenRequestTest.java [hunk 2]
```diff
@@ -78,8 +83,25 @@ public void shouldAddAnAttributeOnlyOnce() {
...
+    @Test
+    public void shouldAllowSameAttributeWithDifferingDerivationNames() {
+        SandboxAttribute derivationAttribute = SandboxAttribute.builder()
+                .withName(SOME_ATTRIBUTE_NAME)
+                .withDerivation("derivation1")
+                .build();
+
+        YotiTokenRequest yotiTokenRequest = YotiTokenRequest.builder()
+                .withAttribute(SOME_ATTRIBUTE)
+                .withAttribute(derivationAttribute)
```
Comment (sidibos) [inline yoti-sdk-sandbox/src/test/java/com.yoti.api.client.sandbox/profile/request/YotiTokenRequestTest.java]: Will the derived attributes have anchors?

Comment (bucky-boy) [reply yoti-sdk-sandbox/src/test/java/com.yoti.api.client.sandbox/profile/request/YotiTokenRequestTest.java]: Yes.  My understanding is that they will come along with all the _anchors_ from the original _date_of_birth_ attribute.

**Comment by bucky-boy** [comment]
After speaking to @qzagarese, we'll go ahead and merge this.  More detail in ticket.

**Pull request merged**